admin-dev/themes/new-theme/.eslintrc.js

```diff
@@ -0,0 +1,32 @@
+// http://eslint.org/docs/user-guide/configuring
+
+module.exports = {
+  env: {
+    browser: true,
+    commonjs: true,
+    node: true,
+    es6: true,
+    jquery: true,
+  },
+  globals: {
+    google: true,
+    document: true,
+    navigator: false,
+    window: true,
+  },
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module',
+  },
+  root: true,
+  extends: 'prestashop',
+  plugins: [
+    'import',
+    'html',
+  ],
+  rules: {
```

Comment:
Don't forget to remove it if you don't use it at the end of your PR

Reply:
Yes, it's a Draft :D